browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2679,6 +2685,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_WALLET_DEX_AGGREGATOR" desc="Brave wallet msg about using 0x for swap">
       Brave uses 0x as a DEX aggregator.
     </message>
+    <message name="IDS_SWAP_DEX_AGGREGATOR_NAME" desc="Brave wallet swap dex aggregator source name">
+      0x
```

Comment:
Please add `translateable="false"` as we don't want 0x to be translated.

Reply:
Got it, this is being used with [indexOf to find the position](https://github.com/brave/brave-core/pull/11978/files#diff-30eae7637d039986bb60e7d56d19a411894f2bce4725def3d4dd0945ea5edc72R1345) in `IDS_WALLET_DEX_AGGREGATOR` so do I need to wrap `0x` in `ph` tag as well, like 
` Brave uses <ph>0x</ph> as a DEX aggregator.`? or not

Reply:
yeah, let's do how it looks like on desktop